_data-prepper/simple-pipelines.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Simple pipelines
+nav_order: 12
+---
+
+# Simple pipelines
+
+This tutorial walks you through creating a simple Data Prepper pipeline and how to configure it.
+
+## Installation
+
+First, install via Docker using the following command:
```

Comment:
suggestion to remove latinism "via->with" 
"install the data prepper with Docker. . .